srcjs/helpers/helpers-disconnect.js

```diff
@@ -9,21 +9,21 @@ export const setCustomDisconnect = (app) => {
         }
       };
     });
-  
+
     $(document).on("shiny:disconnected", function(event) {
       let reconnectToast = app.toast
         .create({
           position: "center",
           text:
-            'Oups... disconnected </br> </br> <div class="row"><button onclick="Shiny.shinyapp.reconnect();" class="toast-button button color-green col">Reconnect</button><button onclick="location.reload();" class="toast-button button color-red col">Reload</button></div>'
+            'Oops... disconnected </br> </br> <button onclick="Shiny.shinyapp.reconnect();" style="margin:0" class="toast-button button color-green col">Reconnect</button><button onclick="location.reload();" style="margin:0" class="toast-button button color-red col">Reload</button>'
         })
         .open();
-  
-      $('.toast').css("background-color", "#1c1c1d");
-  
+
+      $('.toast').css("z-index", "99999");
+
       // close toast whenever a choice is made ...
       $(".toast-button").on("click", function() {
         reconnectToast.close();
       });
     });
-  }; 
+  };
```

Comment:
@DivadNojnarg  perhaps a dialog is even better in this case? It was nice with the icon, but that was a hassle because of the icon deps that needed to be attached I guess.

Reply:
The dialog buttons are more `ok`/ `cancel`. Here we have 2 options but maybe a dialog would render better. You can try and show how it looks like.